lib/renderer/chart_painter.dart

```diff
@@ -403,6 +432,62 @@ class ChartPainter extends BaseChartPainter {
     tp.paint(canvas, Offset(offsetX, top));
   }
 
+//For TrendLine
+  void drawTrendLines(Canvas canvas, Size size) {
+    var index = calculateSelectedX(selectX);
+    Paint paintY = Paint()
+      ..color = Colors.orange
+      ..strokeWidth = 1
+      ..isAntiAlias = true;
+    double x = getX(index);
+    afzlx = x;
```

Comment:
Where use 'afzlx'?

Reply:
Sorry for using my own name as variable name. It was roughly used by me nearly four month ago when I was adding trendline functionality in your package. But now you have upgraded your package. So I forked it yesterday (Because it got new updates) and copied code lines from my old flutter app.

Basicall afzlx is global variable to store value of x in this function and then it is returned in a global function as shown below:
![image](https://user-images.githubusercontent.com/64379149/147353879-11b4f97b-38ef-445f-98f2-f56c3b4efaed.png)

afzl function is called in main file. as shown below:
![image](https://user-images.githubusercontent.com/64379149/147354046-c533c05d-31d1-4080-a5f8-6a2fb8276798.png)

It has been passed a long time since I edited your package So I spent nearly two hours yesterday to copy lines and make them fully functional here. If you do not like afzl you can read and rename according to you. Sorry for poor documentaion. It was not intented to merge it in you branch at that time.